packages/itwin/reports-config-widget/src/widget/components/ReportsRouter.tsx

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { useActiveIModelConnection } from "@itwin/appui-react";
+import React from "react";
+import { ReportsConfigWidget } from "../../ReportsConfigWidget";
+import { ReportAction } from "./ReportAction";
+import { ReportMappings } from "./ReportMappings";
+import { Reports } from "./Reports";
+import type { Route } from "./ReportsContainer";
+import { RouteStep } from "./ReportsContainer";
+
+export const ReportsRouter = ({
+  currentRoute,
+  navigateTo,
+  goBack,
+}: {
+  currentRoute: Route;
+  navigateTo: (toRoute: (prev: Route | undefined) => Route) => void;
+  goBack: () => void;
+}) => {
+  goBack;
```

Comment:
What is this for?

Reply:
Nothing just a noop. Was a debug attempt. I removed it earlier. Pushed my changes.